controllers/calendar.js

```diff
@@ -30,7 +30,9 @@ const mapEventProps = (event) => {
     event.summary = event.attributes.summary;
     event.title = event.attributes.summary;
     event.location = event.attributes.location;
-    event.description = "Test";
+    event.description = event.attributes.description;
```

Comment:
It might be useful to just assign all `event.attributes` properties to `event` automatically